### ISSUE TITLE
unable to delete amplify apps

### ISSUE THREAD

**Issue by mrudulp** [aws-amplify/amplify-console#2566]
Hi,

I am unable to delete AWS Applify apps created while following [tutorial](https://aws.amazon.com/getting-started/guides/deploy-webapp-amplify/module-one/).

I have looked up with a related thread (https://github.com/aws-amplify/amplify-console/issues/1330) and tried things suggested on the thread.
I have deleted CloudFormation Stacks and S3 Buckets but that has not helped.
Also tried to `aws amplify delete-backend-environment --app-id <appid>--environment-n
ame <envname> --profile <profilename>` and that has not helped.
Tried to run `amplify delete` as well.
Nothing seems to help nor going to console and deleting the apps.

Can you please delete following apps that I can still see in my console

Apps to delete
arn:aws:amplify:eu-west-1:085517012113:apps/d3g1n8b3zor650
arn:aws:amplify:eu-west-1:085517012113:apps/dwr18zueaan71
arn:aws:amplify:eu-west-1:085517012113:apps/d3btjeffymf5c
arn:aws:amplify:eu-west-1:085517012113:apps/d4jejc395cz8i

**Comment by xiaoxiaosaohuo** [comment]
yeah, I have this problem too, so strange, I also have tried all the solutions, however, all of them are not helped. Actually, when I delete the app in the console, the remove api status code is 404 ,`https://amplifybackend.ap-northeast-1.amazonaws.com/backend/d2b3nfswdza4da/remove`

**Comment by hloriana** [comment]
Hi @mrudulp 👋🏽 thanks for raising this issue. It appears that those have been delete, are they still visible in your console?

**Comment by hloriana** [comment]
@xiaoxiaosaohuo your application also appears to have been deleted. Can you confirm if they're still showing up in the console?

**Comment by amermoosa** [comment]
yes still showing for me to id [441809585417] its a glitch inside your system fix that without given no meaningful answers by asking still showing or not (westing time) figure out that and solve it  .

**Comment by ganipcanot** [comment]
I'm still seeing `d3g1n8b3zor650` in our system.

Is there any error message displayed in the console or network tab when you try to delete?

**Comment by amermoosa** [comment]
my id is (441809585417) (amermoosa) please delete all application  except (sevenden ) because it was deleted but still show in my console

**Comment by johnpc** [comment]
A recent release fixed several issues preventing apps/backends from being deleted. Please try it again now and let us know if there are still any problems

**Comment by amermoosa** [comment]
thank u for the update it is working now
